src/askui/agent.py

```diff
@@ -61,13 +64,28 @@ def click(self, instruction: Optional[str] = None, button: Literal['left', 'midd
 
         Raises:
             InvalidParameterError: If the 'repeat' parameter is less than 1.
+
+        Example:
+            >>> with VisionAgent() as agent:
+            >>>     agent.click()              # Left click on current position
+            >>>     agent.click("Edit")        # Left click on text "Edit"
+            >>>     agent.click("Edit", button=right)  # Left click on text "Edit"
+            >>>     agent.click(repeat=2)      # Double left click on current position
+            >>>     agent.click("Edit", button="middle", repeat=4)   # 4x middle click on text "edit
         """
         if repeat < 1:
             raise InvalidParameterError("InvalidParameterError! The parameter 'repeat' needs to be greater than 0.")
         self._check_askui_controller_enabled()
+        if self.report is not None:
+            msg = f'click'
+            if button is not 'left':
+                msg = f'{button} ' + msg 
+            if repeat is not 1:
+                msg += f' {repeat}x times'
```

Comment:
msg = 'click'
            if button is not 'left':
                msg = f'{button} {msg}' 
            if repeat != 1:
                msg += f' {repeat}x times'

Reply:
Most important is the `repeat != 1` as otherwise a syntax warning is thrown as we are comparing against a literal.